app/gateway/2.8.x/plan-and-deploy/security/secrets-management/reference-format.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Reference Format
+beta: true
+---
+
+## Reference Format
+
+We use the [URL syntax](https://en.wikipedia.org/wiki/URL) to describe references to a secret store.
+
+```text
+{vault://<vault-backend|entity>/<secret-id>[/<secret-key][?query][#fragment]}
```

Comment:
fragment isn't used, so shouldn't be in the docs. Only once it is used document it